core/templates/components/skill-selector/merge-skill-modal.controller.spec.ts

```diff
@@ -41,23 +41,16 @@ describe('Merge Skill Modal Controller', function() {
     });
   }));
 
-  it('should check properties set after controller is initialized', function() {
-    expect($scope.skillSummaries).toBe(skillSummaries);
-    expect($scope.categorizedSkills).toBe(categorizedSkills);
-    expect($scope.allowSkillsFromOtherTopics).toBe(true);
-    expect($scope.selectedSkillId).toBe('');
-  });
-
-  it('should close modal on saving', function() {
-    $scope.save();
-    expect($uibModalInstance.close).toHaveBeenCalledWith({
-      skill: skill,
-      supersedingSkillId: ''
+  it('should initialize $scope properties after controller is initialized',
+    function() {
+      expect($scope.skillSummaries).toBe(skillSummaries);
+      expect($scope.categorizedSkills).toBe(categorizedSkills);
+      expect($scope.allowSkillsFromOtherTopics).toBe(true);
+      expect($scope.selectedSkillId).toBe('');
     });
-  });
 
-  it('should close modal on confirm', function() {
-    $scope.confirm();
+  it('should close modal on save skill merged', function() {
```

Comment:
This still doesn't make sense to me. It's actually the other way around right?

Should merge skills on closing modal

Reply:
Done.